adapters/dmx/dmx.go

```diff
@@ -0,0 +1,259 @@
+package dmx
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"net/url"
+)
+
+type DmxAdapter struct {
+	endpoint    string
+	publisherId string
```

Comment:
I don't see `publisherId` in this struct being used for anything? It should not belong there anyway, since there will be only one instance of the `DmxAdapter` across all requests, and I imagine that `publisherId` is more dynamic than that.